src/Core/Utilities/OfficeColorUtilities.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.FluentUI.AspNetCore.Components;
+
+public static class OfficeColorUtilities
+{
+    public static OfficeColor GetRandom(bool skipDefault = true)
```

Comment:
To avoid computing all the Colors each time, you could have a code like this:
```csharp
public static readonly DesignThemeModes[] AllThemeModes = Enum.GetValues<DesignThemeModes>();

public static readonly OfficeColor[] AllColors = Enum.GetValues<OfficeColor>();

public static OfficeColor GetRandom(bool skipDefault = true)
{
    var values = AllColors.Skip(skipDefault ? 1 : 0);
    return values.ElementAt(new Random().Next(values.Count()));
}
```